missing-values-imputer/wind-imputation/preprocess.py

```diff
@@ -0,0 +1,31 @@
+from ydata.dataset import Dataset
+
+def preprocess_data(df: Dataset):
+    assert isinstance(df, Dataset), "Provided arguments should be of Dataset type."
+
+    # Data Types
+    DTYPES = {
+    'station': 'str',
+    'timestamp': 'datetime64[ns]',
+    'speed': 'float',
+    'direction': 'float'
+    }
+
+    # Subset relevant columns
+    df = df.select_columns(list(DTYPES.keys()))
+
+    # Convert to pandas and cast to correct data types
+    df = df.to_pandas().astype(DTYPES)
+    
+    # Drop duplicates
+    df = df.drop_duplicates()
+
+    # Sort and set index, sort columns
+    df = df.sort_values(by=['timestamp', 'station']).set_index('timestamp')
```

Comment:
It seems that we could call the method found in the `missing-values-imputer/wind-imputation/utils.py` directly in this method. Wdyt?